src/components/Breadcrumb/Breadcrumb.vue

```diff
@@ -38,7 +38,11 @@ This component is meant to be used inside a Breadcrumbs component.
 		@dragover.prevent="() => {}"
 		@dragenter="dragEnter"
 		@dragleave="dragLeave">
-		<element :is="tag" :to="to" :href="href">
+		<element
+			:is="tag"
+			:to="to"
+			:href="href"
+			:style="[$slots.default ? {paddingRight: '2px'} : {}]">
```

Comment:
I prefer adding specific classes for this rather than inline styling :)

We usually use BEM class structure, so `.crumb--with-action` ?

Reply:
Alright, will change that.